react/Button/Readme.md

```diff
@@ -4,6 +4,7 @@ Both look exactly the same, they share the same `theme`, `className` & `onClick`
 
 - `busy` that adds a spinner (default `false`)
 - `disabled` that disable the click event (default `false`)
+- `type`, eg. `submit` or `reset` (default `submit`)
```

Comment:
Could also be `button` but I don't know if you wanted to list them all or if this was just for the example.

Reply:
Yeah no it was just to list a few :) [Related](http://theoatmeal.com/comics/ie) 😛